prov/sm2/src/sm2_init.c

```diff
@@ -124,6 +124,9 @@ int sm2_create(const struct fi_provider *prov, const struct sm2_attr *attr,
 	/* Need to unlock coordinator so that others can add themselves to
 	 * header */
 	sm2_file_unlock(sm2_mmap);
+
+	FI_WARN(&sm2_prov, FI_LOG_EP_CTRL,
```

Comment:
It feels like this should also be an info log, not really  a warning.

Reply:
I like it as a warn so you can see that you are using SM2 from FI_WARN.... EFA has a very similar warning.  Via FI_WARN, with EFA + SM2, you can see that both are being used without going to INFO.

Reply:
ok

Reply:
But I do hear you that we should treat FI_WARN like an error log to make it as little noisy as possible.

Reply:
Going to keep this is warn for now and maybe move to info in the future.